fig/python-zero-index.svg

```diff
@@ -21,55 +22,78 @@
 <path class="st2" d="M62.8,70.2L62.8,70.2c-4.4,0-8-3.6-8-8V30.2c0-4.4,3.6-8,8-8h0c4.4,0,8,3.6,8,8v32.1
 	C70.8,66.6,67.2,70.2,62.8,70.2z"/>
 <rect x="53.4" y="26" class="st3" width="17.4" height="39.9"/>
-<text transform="matrix(1 0 0 1 57.754 34.6281)"><tspan x="0" y="0" class="st4 st5 st6">A</tspan><tspan x="0" y="14.4" class="st7 st5 st6">D</tspan><tspan x="-0.3" y="28.8" class="st8 st5 st6">G</tspan></text>
+<text transform="matrix(1 0 0 1 57.754 34.6281)" font-weight="bold"><tspan x="0" y="0" class="st4 st5 st6">A</tspan><tspan x="0" y="14.4" class="st7 st5 st6">D</tspan><tspan x="-0.3" y="28.8" class="st8 st5 st6">G</tspan></text>
 <rect x="71.4" y="26" class="st3" width="17.4" height="40.7"/>
-<text transform="matrix(1 0 0 1 75.754 34.6281)"><tspan x="0" y="0" class="st4 st5 st6">B</tspan><tspan x="0.3" y="14.4" class="st7 st5 st6">E</tspan><tspan x="0" y="28.8" class="st8 st5 st6">H</tspan></text>
+<text transform="matrix(1 0 0 1 75.754 34.6281)" font-weight="bold"><tspan x="0" y="0" class="st4 st5 st6">B</tspan><tspan x="0.3" y="14.4" class="st7 st5 st6">E</tspan><tspan x="0" y="28.8" class="st8 st5 st6">H</tspan></text>
 <rect x="89.4" y="26" class="st3" width="17.4" height="38.6"/>
-<text transform="matrix(1 0 0 1 93.754 34.6281)"><tspan x="0" y="0" class="st4 st5 st6">C</tspan><tspan x="0.7" y="14.4" class="st7 st5 st6">F</tspan><tspan x="2.7" y="28.8" class="st8 st5 st6">I</tspan></text>
+<text transform="matrix(1 0 0 1 93.754 34.6281)" font-weight="bold"><tspan x="0" y="0" class="st4 st5 st6">C</tspan><tspan x="0.7" y="14.4" class="st7 st5 st6">F</tspan><tspan x="2.7" y="28.8" class="st8 st5 st6">I</tspan></text>
 <rect x="0" y="26" class="st3" width="53.4" height="45.7"/>
-<text transform="matrix(1 0 0 1 7.0493 34.6281)"><tspan x="0" y="0" class="st5 st6">data = [</tspan><tspan x="42.3" y="0" class="st4 st5 st6">[</tspan><tspan x="42.3" y="14.4" class="st7 st5 st6">[</tspan><tspan x="42.3" y="28.8" class="st8 st5 st6">[</tspan></text>
```

Comment:
I see the spaces were moved from around the `=` which goes against the Python convention, though the way this is rendering as a png, I don't think you really see a difference between this and where there are spaces are still in place in `data[0, 0] = A`.

Reply:
thanks for pointing this out!
It may take a little while to get to this since I will need to nudge each element to fit.

Honestly, I would need to figure out how to use some proper svg editing software instead of changing the svg file directly.

If you're alright with the pull request sitting for a while, I'll get to it.

Reply:
> Honestly, I would need to figure out how to use some proper svg editing software instead of changing the svg file directly.

Have you tried using `svgcleaner`?
We have some notes about SVGs here:
https://github.com/swcarpentry/python-novice-inflammation/blob/gh-pages/fig/README.md